packages/turbo/ios/RNSession.swift

```diff
@@ -80,8 +80,12 @@ class RNSession: NSObject {
   func visit(_ visitable: Visitable) {
     turboSession.visit(visitable)
   }
-  
+    
   func reload() {
+    webView.reload()
+  }
+  
+  func refresh() {
```

Comment:
This is not what I think we want. We want the `refresh` to perform a `replace` visit to the current URL, and the `reload` to reload the `turboSession`

Reply:
Ie. something along the lines of:

```swift
func reload() {
  turboSession.reload()
}

func refresh() {
  // turboSession.visit with the current URL and action = replace
}

Reply:
`turboSession.reload()` issues a visit with `topmostVisit` as a parameter:
```
    public func reload() {
        guard let visitable = topmostVisitable else { return }

        initialized = false
        visit(visitable)
        topmostVisit = currentVisit
    }
   ```

I'm wondering if that's what we want - we want to do a coldboot not another visit so let's go with:

```
func reload() {
  webView.reload()
}

func refresh() {
  // turboSession.visit with the current URL and action = replace
}
```

Reply:
I'm pretty sure the `turboSession.reload()` is what we want. It reloads the current page setting `initialized = false` before, which issues a `ColdBoot` visit:

![image](https://github.com/software-mansion-labs/react-native-turbo-demo/assets/195925/0f4e8300-8bd2-4d14-bb64-d234f975a0e8)